scripts/ccpp_prebuild.py

```diff
@@ -639,20 +701,13 @@ def generate_schemes_makefile(schemes, schemes_makefile, schemes_cmakefile, sche
     cmakefile.filename = schemes_cmakefile
     sourcefile = SchemesSourcefile()
     sourcefile.filename = schemes_sourcefile
-    # Adjust relative file path to schemes from caps makefile
-    schemes_with_path = []
-    schemes_with_abspath = []
-    schemes_makefile_dir = os.path.split(os.path.abspath(schemes_makefile))[0]
-    for scheme in schemes:
-        (scheme_filepath, scheme_filename) = os.path.split(os.path.abspath(scheme))
-        relative_path = './{0}'.format(os.path.relpath(scheme_filepath, schemes_makefile_dir))
-        schemes_with_path.append(os.path.join(relative_path, scheme_filename))
-        schemes_with_abspath.append(os.path.abspath(scheme))
+    # Generate list of schemes with absolute path
+    schemes_with_abspath = [ os.path.abspath(scheme) for scheme in schemes ]
```

Comment:
Is this an unrelated improvement?

Reply:
Yes, cleanup (in a second place just below/above as well).